packages/ui-components/style/icons/filetype/console.svg

```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="16" viewBox="0 0 200 200">
-  <g class="jp-icon-brand1 jp-icon-selectable" fill="#0288D1">
+  <g class="jp-console-icon-color jp-icon-selectable" fill="#0288D1">
```

Comment:
We could probably use the same structure as for the terminal icon:
`--jp-console-icon-background-color` for this
`--jp-console-icon-color` for the white parts

Reply:
That's right. `jp-console-icon-background-color`has been added and used in `console.svg` in a similar way to what is done with the terminal icon.